components/ai_chat/core/browser/engine/oai_message_utils_unittest.cc

```diff
@@ -28,6 +28,23 @@ struct RewriteActionTestParam {
   std::string expected_payload;  // non-empty for change tones
 };
 
+EngineConsumer::ConversationHistory BuildTestHistory() {
```

Comment:
can we use `CreateSampleChatHistory` in `components/ai_chat/core/browser/test_utils.cc`?